cmd/rofl/build/artifacts.go

```diff
@@ -359,36 +359,39 @@ func createSquashFs(buildEnv env.ExecEnv, fn, dir string) (int64, error) {
 	tarHash := hex.EncodeToString(tarHasher.Sum(nil))
 	fmt.Printf("TAR archive SHA256: %s\n", tarHash)
 
+	// Convert paths for container environment if needed.
+	fnInEnv, err := buildEnv.PathToEnv(fn)
+	if err != nil {
+		return 0, fmt.Errorf("failed to convert output path: %w", err)
+	}
+	tarPathInEnv, err := buildEnv.PathToEnv(tarPath)
+	if err != nil {
+		return 0, fmt.Errorf("failed to convert tar path: %w", err)
+	}
+
 	// Convert tar to squashfs using sqfstar under fakeroot.
+	// We use shell redirection instead of cmd.Stdin because stdin doesn't get piped through
+	// Docker when running in a container environment.
```

Comment:
Hm isn't that strange, maybe it needs a flag to pipe?

Reply:
You were right, we just needed `-i` for docker. Updated, since that's much nicer fix. Thanks!